### DISCUSSION TITLE
ROOT6 suport for Intel Mac on Sequoia

### DISCUSSION THREAD

**Discussion by chaen0120** [Homebrew/discussions#5734] [Everyday usage]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

The ROOT6 package provided does not support macOS 15.1 on my Intel-based Mac.
Do you have any plans to update it?

**Comment by Bo98** [comment]
What issues are you seeing when trying it on macOS 15.1? Generally speaking, we're only building macOS 15 packages for those that need it - the macOS 14 ones are used for those that don't have dedicated macOS 15 bottles and work fine for 99%+ of packages.

**Reply by chaen0120** [reply]
When I try `brew install root`, it seems to install from the package for MacOS 14. 

> ==> Installing root
==> Pouring root--6.32.08.sonoma.bottle.tar.gz

And it doesn't work after the installation.
> $ root
 *** Break *** illegal instruction

**Reply by carlocab** [reply]
What is the output of `brew config` and `brew doctor`?

**Reply by chaen0120** [reply]
$brew config
HOMEBREW_VERSION: 4.4.5-35-gcd8f476
ORIGIN: https://github.com/Homebrew/brew
HEAD: cd8f476ffe6edd429a435a55c10d9928fed201ce
Last commit: 2 hours ago
Core tap HEAD: 75c0a69cda3c0551977a54acd678b2c2d4b680cf
Core tap last commit: 2 weeks ago
Core tap JSON: 15 Nov 02:11 UTC
Core cask tap HEAD: b089401a11c647cefed77424d399d79c00063a6d
Core cask tap last commit: 2 weeks ago
Core cask tap JSON: 15 Nov 02:11 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.Bw8IlWWYt6/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.6 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.6/bin/ruby
CPU: octa-core 64-bit kabylake
Clang: 16.0.0 build 1600
Git: 2.39.5 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.1-x86_64
CLT: 16.1.0.0.1.1729049160
Xcode: 16.1

$brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have the following deprecated, cask taps tapped:
  caskroom/cask
Untap them with `brew untap`.

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libCLHEP-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-Cast-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-Cast-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-Evaluator-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-Evaluator-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-Exceptions-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-Exceptions-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-GenericFunctions-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-GenericFunctions-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-Geometry-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-Geometry-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-Matrix-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-Matrix-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-Random-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-Random-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-RandomObjects-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-RandomObjects-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-RefCount-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-RefCount-2.4.4.0.dylib
  /usr/local/lib/libCLHEP-Vector-2.4.1.3.dylib
  /usr/local/lib/libCLHEP-Vector-2.4.4.0.dylib
  /usr/local/lib/libvdt.dylib

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae and may need to be deleted.

Unexpected header files:
  /usr/local/include/ROOT-RELATED-HEADERS

Warning: Unbrewed '.pc' files were found in /usr/local/lib/pkgconfig.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae and may need to be deleted.

Unexpected '.pc' files:
  /usr/local/lib/pkgconfig/clhep-cast.pc
  /usr/local/lib/pkgconfig/clhep-evaluator.pc
  /usr/local/lib/pkgconfig/clhep-exceptions.pc
  /usr/local/lib/pkgconfig/clhep-genericfunctions.pc
  /usr/local/lib/pkgconfig/clhep-geometry.pc
  /usr/local/lib/pkgconfig/clhep-matrix.pc
  /usr/local/lib/pkgconfig/clhep-random.pc
  /usr/local/lib/pkgconfig/clhep-randomobjects.pc
  /usr/local/lib/pkgconfig/clhep-refcount.pc
  /usr/local/lib/pkgconfig/clhep-vector.pc
  /usr/local/lib/pkgconfig/clhep.pc

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libCLHEP-2.4.1.3.a
  /usr/local/lib/libCLHEP-2.4.4.0.a
  /usr/local/lib/libCLHEP-Cast-2.4.1.3.a
  /usr/local/lib/libCLHEP-Cast-2.4.4.0.a
  /usr/local/lib/libCLHEP-Evaluator-2.4.1.3.a
  /usr/local/lib/libCLHEP-Evaluator-2.4.4.0.a
  /usr/local/lib/libCLHEP-Exceptions-2.4.1.3.a
  /usr/local/lib/libCLHEP-Exceptions-2.4.4.0.a
  /usr/local/lib/libCLHEP-GenericFunctions-2.4.1.3.a
  /usr/local/lib/libCLHEP-GenericFunctions-2.4.4.0.a
  /usr/local/lib/libCLHEP-Geometry-2.4.1.3.a
  /usr/local/lib/libCLHEP-Geometry-2.4.4.0.a
  /usr/local/lib/libCLHEP-Matrix-2.4.1.3.a
  /usr/local/lib/libCLHEP-Matrix-2.4.4.0.a
  /usr/local/lib/libCLHEP-Random-2.4.1.3.a
  /usr/local/lib/libCLHEP-Random-2.4.4.0.a
  /usr/local/lib/libCLHEP-RandomObjects-2.4.1.3.a
  /usr/local/lib/libCLHEP-RandomObjects-2.4.4.0.a
  /usr/local/lib/libCLHEP-RefCount-2.4.1.3.a
  /usr/local/lib/libCLHEP-RefCount-2.4.4.0.a
  /usr/local/lib/libCLHEP-Vector-2.4.1.3.a
  /usr/local/lib/libCLHEP-Vector-2.4.4.0.a

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  libcerf
  dbus
  double-conversion
  kubernetes-cli
  fftw

Warning: You have uncommitted modifications to Homebrew/homebrew-core.
If this is a surprise to you, then you should stash these modifications.
Stashing returns Homebrew to a pristine state but can be undone
should you later need to do so for some reason.
  cd /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core && git stash -u && git clean -d -f

Uncommitted files:
  ?? Formula/root.rb

**Reply by cho-m** [reply]
ROOT does bake in particular SDK path. Maybe going back to full Xcode.app dependency could help, though ROOT has broken on new Xcode.app releases before.
